examples/plot_kernel_approximation.py

```diff
@@ -57,6 +57,80 @@
 from sklearn.kernel_approximation import (RBFSampler,
                                           Nystroem)
 from sklearn.decomposition import PCA
+from collections import defaultdict
+
+
+def timing(callable, *args, **kwargs):
+    """Time the call of a function and return time and
+    result. Passing args and kwargs to function.
+    """
+    init_time = time()
+    result = callable(*args, **kwargs)
+    return result, time() - init_time
+
+
+def fit_score(clf, train, test):
```

Comment:
If its purpose is only to assist in making `timing` a separate function, I'd rather time only fit or not have either helper.